docs/measurements/procedures/get_bad_bpms.md

```diff
@@ -0,0 +1,148 @@
+# Get a list of Bad BPMs
+
+Every now and then BI asks for a list of BPMs that cause trouble for our measurements so that they can take a closer look during the next [(E?YET|L)S](https://regex101.com/r/ShN5hC/3).
```

Comment:
I don't feel like a hyperlink to `regex101` is warranted, I'd rather see a simple admonition pop up explaining the meaning of `YETS`.

Reply:
ooooh, I liked this little joke